NUnitTests/ABS_Pos_Test.cs

```diff
@@ -0,0 +1,24 @@
+using CSharpCalculator;
+
+namespace NUnitTests;
+
+public class ABS_Pos_Test
+{
+    double x;
+    double expected;
+    
+    [SetUp]
+    public void Setup()
+    {
+        x = -5;
```

Comment:
In the case when we have only one testing method and one set of test parameters, then an approach like yours will do. But if we have more test parameters, then I recommend moving the test data into an attribute (as shown in the lesson).